### PR TITLE
Add Chinese translation for store

### PR DESCRIPTION
**Pull request by EliseRoland**
add file:
Dialog_NPC_Shop_Customer_CN.txt
Dialog_NPC_Shop_Vendor_CN.txt
Text_Shop_CN.txt

update CN translation dictionary in BondageClub/Scripts/Translation.js

Fix bug a in BondageClub\Screens\Room\MaidQuarters\Dialog_NPC_MaidQuarters_Maid_CN.txt

### CODE REVIEW

File: BondageClub/Screens/Room/Shop/Dialog_NPC_Shop_Vendor_CN.txt [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+You've took care of everything.  Please send my regards to the maids of the sorority.
+你已经处理好了一切。请代我向女生联谊会的女仆们问好
+###_PLAYER
+What happened here?
+发生什么事了？
+###_NPC
+I've made a deal with a client that if she bought everything, she could use the items on me.  So, she did just that.
+我和一位客户达成了约定，如果她买了店里的所有东西，她就可以把这些东西用在我身上。所以，她就这么做了
+A burglar surprised and restrained me.  The maids caught her and dragged her away.  I guess they sent you to help me.
+一个窃贼吓了我一跳，还我束缚住了。女仆们抓住她，把她拖走了。我猜他们派你来帮我
```
Comment (remiliacn) [inline BondageClub/Screens/Room/Shop/Dialog_NPC_Shop_Vendor_CN.txt:361]:
```suggestion
一个窃贼吓了我一跳，还把我束缚住了。女仆们抓住她，把她拖走了。我猜他们派你来帮我
```

File: BondageClub/Screens/Room/Shop/Dialog_NPC_Shop_Vendor_CN.txt [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+我很高兴为你服务，小姐。
+###_NPC
+You maids are so sweet.  Would you like to buy something?
+你们这些女孩可真可爱。你想买点什么吗？
+###_PLAYER
+Will you give me a discount?
+你能给我点折扣吗？
+###_NPC
+Nice try, but I can't do that.  The sorority will pay you for your work though.  Would you like to buy something?
+很好的尝试，但我不能那么做。不过女生联谊会会会为你的工作付钱。你想买点什么吗？
```
Comment (remiliacn) [inline BondageClub/Screens/Room/Shop/Dialog_NPC_Shop_Vendor_CN.txt:379]:
```suggestion
很好的尝试，但我不能那么做。不过女仆联谊会会会为你的工作付钱。你想买点什么吗？
```

File: BondageClub/Screens/Room/Shop/Dialog_NPC_Shop_Vendor_CN.txt [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+###_PLAYER
+The sorority will always protect you.
+女生联谊会会永远保护你
```
Comment (remiliacn) [inline BondageClub/Screens/Room/Shop/Dialog_NPC_Shop_Vendor_CN.txt:382]:
```suggestion
女仆联谊会会永远保护你
```

File: BondageClub/Screens/Room/Shop/Dialog_NPC_Shop_Vendor_CN.txt [hunk 4]
```diff
@@ -0,0 +1,385 @@
...
+Items for hands.
+用在手上的东西
+Items for breasts.
+用在胸上的东西
+Toys for the nipples.
+乳头玩具
+Items to decorate the nipples.
+乳头装饰品
+Harnesses for the torso.
+躯干安全带
```
Comment (remiliacn) [inline BondageClub/Screens/Room/Shop/Dialog_NPC_Shop_Vendor_CN.txt:316]: 目前翻译是`束缚带`，改一下？安全带不是很恰当感觉
```suggestion
躯干束缚带
```

**Review by remiliacn — CHANGES_REQUESTED** [review]
I have a few comments, other than that, LGTM.

**Comment by Ben987** [comment]
Very nice!  Thanks a lot for that translation :)

**Pull request merged**